data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,280 @@ event "remnant: pantica void sprites"
+
+
+
+event "remnant: lab availability"
+	outfitter "Remnant"
+		add "Research Laboratory"
+
+
+
+mission "Remnant: Expanded Horizons Astral 2"
+	name "Research expedition to Sagittarius A*"
+	description "Travel to Sagittarius A* with a research laboratory installed in order to make measurements on the supermassive black hole and then return to <destination>."
+	blocked "You need at least 1 free bunk, a jump drive, and a research laboratory to take this mission."
+	waypoint "Sagittarius A*"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Astral 1: done"
+	on offer
+		require "Jump Drive"
+		require "Research Laboratory"
+		conversation
+			`Dawn is just finishing her meal when you enter the spaceport cafeteria. She stands up as you approach and deposits the tray in a receptacle next to her table. "Well, ready to go?"`
+			choice
+				`	"Yes."`
+					goto astralonwards
+				`	"I should double check everything."`
+					defer
+			`	She nods. "I am ready, so I will come with you and wait onboard the <ship> while you do your checks."`
+			label astralonwards
+			`	She gestures towards the exit closest to your ship and heads off at a quick pace. Outside, she moves calmly through the bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns and waits for you to catch up.`
+			`	Just as you arrive and reach for the console to open the airlock, you sense a change in your surroundings. Looking around, you see that a group of Remnant gathered around the two of you, focused on Dawn. A young Remnant - scarcely more than a child - steps forward. "Thank you for your patience," he says, then steps back. An elderly Remnant steps forward. "Thank you for your curiosity," she says, then steps back. A middle-aged Remnant steps forward. "Go with our blessing, Seeker, and know that wherever you go, the Remnant go with you." She says it simply, but you feel the weight of tradition behind them. She turns to you. "And you, stranger who walks the stars... May the Embers guide your steps as you walk among us." She nods respectfully, then steps back.`
+			`	As soon as the third steps back there is a flurry of movement as each of the Remnant present quickly step forward, give her a quick hug, and then scurry off into the starport. Before you process what is going on, all the Remnant gathered around you have vanished. You look at Dawn, and her expression is a mixture of awe and humility.`
+			choice
+				`	"They know you are coming back, right?"`
+				`	"What was that about?"`
+				`	"Alright, let's get going."`
+					goto astralmoving
+			`	"Well, what we are doing is dangerous. We will be crossing Korath territory, a wide expanse of uninhabited space, and then scanning a black hole. I have confidence that we will return, but nonetheless there is still a chance that we die in the process.`
```

Comment:
`that we die` doesn't sound right to me.  I think it should be `that we could die`

Reply:
I agree, `there is still a chance that we could die in the process" does sound more natural than "there is still a chance that we die in the process."

That being said, I think it is still technically correct; and I like the slightly more formal/unnatural framing for it. The Player and Dawn are going through a process where death is more than a mere possibility. The way I have it framed makes death sound a bit more likely. It isn't a chance that they will be in a situation where death might occur. They *will* be in a situation where death is a possibility. The unknown factor is whether they die, not whether they are in a situation where they *could* die.

Reply:
What about *will* die, then?

Reply:
That could work. I'll change that over and see how people like it.